packages/cadl-typescript/src/modelUtils.ts

```diff
@@ -693,6 +698,13 @@ function mapCadlIntrinsicModelToTypeScript(
             schema.outputTypeName = `Record<string, ${valueType.name}Output>`;
             schema.outputValueTypeName = `${valueType.name}Output`;
           }
+        } else if (isUnknownType(indexer.value!)) {
+          schema.typeName = `Record<string, ${valueType.type}>`;
+          schema.valueTypeName = valueType.name;
```

Comment:
Is this valueTypeName used for imported models ? If yes, I am not sure what we should import if the type is Record<string, unknown> or Record<string, any>